### ISSUE TITLE
Pls tell me how create a newone

### ISSUE THREAD

**Issue by johnfelipe** [konstantint/talkofacta#7]
Hi, me again pls tell me how integrate new one....
I have this http://comisionsextacamara.jhonfelipeurregomejia.com/, and create xls file with table, but i want create them

Tnks

**Comment by konstantint** [comment]
- read the `README` in the project root dir.
- study the `compute_results.sh` in the project dir.
- study the code of [csv2db.py](https://github.com/konstantint/talkofacta/blob/master/src/talkofactadb/talkofactadb/scripts/csv2db.py), which is the script for converting your CSV file to the database.
- prepare the `acta.csv.gz` file which corresponds to what is read by `csv2db` (apparently it has to have 4 columns: hansard, speaker, speech, date).
- then follow the instructions in `README` and `compute_results` to compute the results into a `resultsdb.sqlite`.
- once it's done you need to put the resulting `sqlite` file into `src/talkofactaweb/data`, and then deploy the application following the instructions in `src/talkofactaweb/README`.

**Comment by johnfelipe** [comment]
Pls give me a hand i send you new csv to your email
Stay alert

**Comment by konstantint** [comment]
I'm rather busy at the moment and I'm not sure I have a lot of time to work on it at the moment. You can send your data, but I can't promise I'll find much time within the next weeks for it.

**Comment by johnfelipe** [comment]
hi please give me help for new csv

**Comment by johnfelipe** [comment]
pls tell me if u can help me this weekend

**Comment by konstantint** [comment]
No, sorry. I really am rather busy at the moment. Don't you have anyone around who can run some python code according to instructions? All the needed steps are documented, and I'm happy to answer specific questions if something is not clear.
